openbb_terminal/miscellaneous/scripts/test_stocks.openbb

```diff
@@ -1,9 +1,13 @@
 stocks
-search apple
+search -c United States -s Consumer Cyclical -i Specialty Retail -e USA -l 5
+load aaple -i 5
 load aapl -s 2022-10-01
+quote
 tob
 tob -e EDGX
-candle
+candle -t --ma 5,10
```

Comment:
Should we keep commands with default parameters AND commands with additional flags/arguements?

Reply:
Probably depends. Running `candle` runs the same code as `candle -t --ma 5,10` just skipping those flags, so if this fails, you are correct that we would not know if its because default pars, the -t par or the --ma par.